src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer(String name) {
+        super(name);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started its work");
```

Comment:
```suggestion
        System.out.println("Bulldozer started its work");
```
Change it in each class

Reply:
this was done for scalability

Reply:
It was not the task here

Reply:
Changed it in all classes